### ISSUE TITLE
TypeError: e.join is not a function

### ISSUE THREAD

**Issue by josdejong** [edravis/logatim#15]
I tried to use logatim (nodejs v5.5) but it gives an error:

```
$ npm install logatim
$ node
> const logatim = require('logatim');
undefined
> logatim.warn('oops')
TypeError: e.join is not a function
    at a (/home/jos/tmp/node_modules/logatim/dist/logatim.umd.min.js:1:3955)
    at i (/home/jos/tmp/node_modules/logatim/dist/logatim.umd.min.js:1:3818)
    at s (/home/jos/tmp/node_modules/logatim/dist/logatim.umd.min.js:1:3729)
    at repl:1:9
    at REPLServer.defaultEval (repl.js:252:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:417:12)
    at emitOne (events.js:95:20)
    at REPLServer.emit (events.js:182:7)
> 
```

Am I overlooking something obvious?

**Comment by sospedra** [comment]
Damn! You're right. While making the styles I forget to check if the output without styling was good.
Really appreciate it. Now it's fixed here cd56fdf34e531d445010b9dbc3224317edda969d

I'm closing the issue since the new tests are passing. However, feel free to comment anything related to this ;)

**Comment by josdejong** [comment]
Thanks for the quick fix!
